yogstation/code/modules/surgery/organs/heart.dm

```diff
@@ -0,0 +1,22 @@
+/obj/item/organ/heart/nanite
+	name = "Nanite heart"
+	desc = "A specialized heart constructed from nanites that helps coordinate nanites allowing them to regenerate quicker inside the body without any ill effects. Caution this organ will fall apart without nanites to sustain itself!"
+	icon_state = "heart-x"
+	organ_flags = ORGAN_SYNTHETIC
+	var/nanite_boost = 1
+
+/obj/item/organ/heart/nanite/emp_act()
+	. = ..()
+	if(. & EMP_PROTECT_SELF)

```

Comment:
What is the point of this?

Reply:
since nanite isnt in the heart/cybernetic tree I just copied this so It would remove nanites when you get emp'ed but im kinda new to this so would the same thing be accomplished without if(. & EMP_PROTECT_SELF)